tools/slurm/scripts/slurmcluster.sh

```diff
@@ -214,16 +206,6 @@ if [[ $OPT_CONTAINER_RUN_TYPE == "enroot" ]]; then
     fi
 fi
 
-TEMPYAML=$TEMPDIR/slurmcluster.yaml
-envsubst <$PARENT_PATH/slurmcluster.yaml >$TEMPYAML
-if [[ -n $DETERMINED_AGENT ]]; then
```

Comment:
I'm curious here what deploys the agents? As in is there another step which launches the agents? I'm not super familiar with this code

Reply:
I am myself not very familiar with this shell script, but according to these lines I understand that -` If $DETERMINED_AGENT is set, this command deletes lines between resource_manager and resource_manager_end in $TEMPYAML.` 
But we do not need to generate a devcluster file deployed with agent anymore.

Reply:
ah ok, looks like agents are deployed manually so we don't need to do anything else